gammapy/catalog/hawc.py

```diff
@@ -0,0 +1,155 @@
+
+        ss = ''
+        ops = info.split(',')
+        if 'basic' in ops:
+            ss += self._info_basic()
+        if 'position' in ops:
+            ss += self._info_position()
+        if 'spectrum' in ops:
+            ss += self._info_spectrum()
+
+        return ss
+
+    def _info_basic(self):
+        """Print basic info."""
+        d = self.data
+        ss = '\n*** Basic info ***\n\n'
+        ss += 'Catalog row index (zero-based) : {}\n'.format(d['catalog_row_index'])
+        ss += '{:<15s} : {}\n'.format('Source name:', d['source_name'])
+
+        return ss
+
+    def _info_position(self):
+        """Print position info."""
+        d = self.data
+        ss = '\n*** Position info ***\n\n'
+
+        ss += 'Measurement:\n'
+        ss += '{:20s} : {:.3f}\n'.format('RA', d['ra'])
+        ss += '{:20s} : {:.3f}\n'.format('DEC', d['dec'])
+        ss += '{:20s} : {:.3f}\n'.format('GLON', d['glon'])
+        ss += '{:20s} : {:.3f}\n'.format('GLAT', d['glat'])
+        ss += '{:20s} : {:.3f}\n'.format('Position error', d['pos_err'])
+
+        return ss
+
+    def _info_spectrum(self):
+        """Print spectral info."""
+        d = self.data
+        ss = '\n*** Spectral info ***\n\n'
+
+        ss += 'Spectrum 1:\n'
+        ss += '{:20s} : {:.3} +- {:.3} {}\n'.format('Flux at 7 TeV', d['spec0_dnde'].value, \
+            d['spec0_dnde_err'].value, 'cm-2 s-1 TeV-1')
+        ss += '{:20s} : {:.3f} +- {:.3f}\n'.format('Spectral index', d['spec0_index'], d['spec0_index_err'])
+        ss += '{:20s} : {:1}\n\n'.format('Test radius', d['spec0_radius'])
+
+        if(np.isnan(d['spec1_dnde'])):
+            ss += 'No second spectrum available for this source'
+        else:
+            ss += 'Spectrum 2:\n'
+            ss += '{:20s} : {:.3} +- {:.3} {}\n'.format('Flux at 7 TeV', d['spec1_dnde'].value, \
+                d['spec1_dnde_err'].value, 'cm-2 s-1 TeV-1')
+            ss += '{:20s} : {:.3f} +- {:.3f}\n'.format('Spectral index', d['spec1_index'], d['spec1_index_err'])
+            ss += '{:20s} : {:1}'.format('Test radius', d['spec1_radius'])
+
+        return ss
+
+    @property
+    def n_spectra(self):
+        """Number of measured spectra (1 or 2)."""
+
+        return 1 if np.isnan(self.data['spec1_dnde']) else 2
+
+    #TODO: Maybe as public method?
+    def _get_spec_pars(self, id):
+        pars, errs = {},{}
+        data = self.data
+
+        pars['amplitude'] = data[f'spec{id}_dnde']
+        errs['amplitude'] = data[f'spec{id}_dnde_err']
+        pars['index'] = data[f'spec{id}_index']*u.Unit('')
+        errs['index'] = data[f'spec{id}_index_err']*u.Unit('')
+        pars['reference'] = 7*u.TeV
+
+        return [pars, errs]
+
+    def spectral_model(self):
+        """Returns a list of `~gammapy.spectrum.models.SpectralModel`
+        objects. Either with one or with two entries depending on the
+        source in the catalog.
+        """
+
+        model_class = PowerLaw
```

Comment:
No need for `model_class = PowerLaw` local variable. Please remove and use `PowerLaw` directly below.